egs/vcc2020/vc1/run_task1.sh

```diff
@@ -372,32 +382,75 @@ if [ ${stage} -le 12 ] && [ ${stop_stage} -ge 12 ]; then
         ark,scp:${outdir}_denorm/${pairname}/feats.ark,${outdir}_denorm/${pairname}/feats.scp
-    convert_fbank.sh --nj ${nj} --cmd "${train_cmd}" \
-        --fs ${fs} \
-        --fmax "${fmax}" \
-        --fmin "${fmin}" \
-        --n_fft ${n_fft} \
-        --n_shift ${n_shift} \
-        --win_length "${win_length}" \
-        --n_mels ${n_mels} \
-        --iters ${griffin_lim_iters} \
-        ${outdir}_denorm/${pairname}/ \
-        ${outdir}_denorm/${pairname}/log \
-        ${outdir}_denorm/${pairname}/wav
-    
-    echo "Generate hdf5"
-    # generate h5 for neural vocoder
-    local/feats2hdf5.py \
-        --scp_file ${outdir}_denorm/${pairname}/feats.scp \
-        --out_dir ${outdir}_denorm/${pairname}/hdf5/
-    (find "$(cd ${outdir}_denorm/${pairname}/hdf5; pwd)" -name "*.h5" -print &) | head > ${outdir}_denorm/${pairname}/hdf5_feats.scp
 
+    # GL
+    if [ ${voc} = "GL" ]; then
+        echo "Using Griffin-Lim phase recovery."
+        convert_fbank.sh --nj ${nj} --cmd "${train_cmd}" \
+            --fs ${fs} \
+            --fmax "${fmax}" \
+            --fmin "${fmin}" \
+            --n_fft ${n_fft} \
+            --n_shift ${n_shift} \
+            --win_length "${win_length}" \
+            --n_mels ${n_mels} \
+            --iters ${griffin_lim_iters} \
+            ${outdir}_denorm/${pairname}/ \
+            ${outdir}_denorm/${pairname}/log \
+            ${outdir}_denorm/${pairname}/wav
+    # PWG
+    elif [ ${voc} = "PWG" ]; then
+        echo "Using Parallel WaveGAN vocoder."
+
+        # variable settings
+        [ -z "${voc_checkpoint}" ] && voc_checkpoint="$(find "${voc_expdir}" -name "*.pkl" -print0 | xargs -0 ls -t | head -n 1)"
+        voc_conf="$(find "${voc_expdir}" -name "config.yml" -print0 | xargs -0 ls -t | head -n 1)"
+        voc_stats="$(find "${voc_expdir}" -name "stats.h5" -print0 | xargs -0 ls -t | head -n 1)"
+        wav_dir=${outdir}_denorm/${pairname}/pwg_wav
+        hdf5_dir=${outdir}_denorm/${pairname}/hdf5
+        hdf5_norm_dir=${outdir}_denorm/${pairname}/hdf5_norm
+        [ ! -e "${wav_dir}" ] && mkdir -p ${wav_dir}
+        [ ! -e ${hdf5_norm_dir} ] && mkdir -p ${hdf5_norm_dir}
+        
+        # generate h5 for neural vocoder
+        local/feats2hdf5.py \
+            --scp_file ${outdir}_denorm/${pairname}/feats.scp \
+            --out_dir ${hdf5_dir} 
+        #(find "$(cd ${hdf5_dir}; pwd)" -name "*.h5" -print &) | head > ${outdir}_denorm/${pairname}/hdf5_feats.scp
+
+        # normalize and dump them
+        echo "Normalizing..."
+        ${train_cmd} --num-threads "${nj}" "${hdf5_norm_dir}/normalize.log" \
+            local/normalize_for_pwg.py \
+                --config "${voc_conf}" \
+                --stats "${voc_stats}" \
+                --rootdir ${hdf5_dir} \
+                --dumpdir ${hdf5_norm_dir} \
+                --n_jobs "${nj}" \
+                --verbose "${verbose}"
+        echo "successfully finished normalization."
```

Comment:
I made new option for this part.
You can use `--skip-wav-copy` option as follows
```bash
parallel-wavengan-normalize \
	--skip-wav-copy \
	--config ${voc_conf} \
	--feats-scp ${outdir}_denorm/${pairname}/feats.scp \
	--dumpdir ${hdf5_norm_dir}
```

Reply:
Note that if the stats are the same in both tts and voc, we can directly use generated feats.scp.